SCClassLibrary/Common/Control/Server.sc

```diff
@@ -388,7 +388,11 @@ Server {
 	*remote { |name, addr, options, clientID|
 		var result;
 		result = this.new(name, addr, options, clientID);
-		result.startAliveThread;
+		if(options.protocol == \tcp) {
+			addr.tryConnectTCP({ result.startAliveThread }, nil, 20)
```

Comment:
looks good!

It might be useful to have a message on failure.

Reply:
It seems that `.tryConnectTCP` does post a warning when it fails:
```WARNING: Couldn't connect to TCP address 127.0.0.1:57110```
Do we need another warning?

Reply:
ah no, that is unnecessary.